src/main/java/de/tum/in/www1/artemis/domain/tutorialgroups/TutorialGroupRegistration.java

```diff
@@ -0,0 +1,68 @@
+package de.tum.in.www1.artemis.domain.tutorialgroups;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+
+import org.hibernate.annotations.Cache;
+import org.hibernate.annotations.CacheConcurrencyStrategy;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import de.tum.in.www1.artemis.domain.DomainObject;
+import de.tum.in.www1.artemis.domain.User;
+import de.tum.in.www1.artemis.domain.enumeration.tutorialgroups.TutorialGroupRegistrationType;
+
+@Entity
+@Table(name = "tutorial_group_registration")
+@Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public class TutorialGroupRegistration extends DomainObject {
+
+    @ManyToOne
+    @JoinColumn(name = "student_id")
+    @NotNull
+    private User student;
+
+    @ManyToOne
+    @JoinColumn(name = "tutorial_group_id")
+    @NotNull
+    private TutorialGroup tutorialGroup;
+
+    @Enumerated(EnumType.STRING)
+    @Column(name = "type")
+    private TutorialGroupRegistrationType type;
+
+    public TutorialGroupRegistration() {
+        // Empty constructor
```

Comment:
For consistency: (Also see my other comment)
```suggestion
        // Empty constructor needed for Jackson.
```